src/Jinja2CppLight.h

```diff
@@ -177,7 +177,10 @@ class Template {
 
 class ControlSection {
 public:
-    std::vector< ControlSection * >sections;
+    
+    virtual ~ControlSection() { sections.clear(); }
```

Comment:
As far as I know, std::vector destructor should do it automatically.

Reply:
I thought so too, but valgrind was reporting leaks until I added the clear.